coldfront/plugins/qumulo/views/update_allocation_view.py

```diff
@@ -11,21 +14,30 @@
     AllocationAttributeChangeRequest,
     AllocationChangeRequest,
     AllocationChangeStatusChoice,
+    AllocationLinkage,
     AllocationUser,
 )
+from coldfront.core.user.models import User
+from coldfront.core.utils.mail import send_email
 from coldfront.plugins.qumulo.forms import UpdateAllocationForm
+from coldfront.plugins.qumulo.hooks import acl_reset_complete_hook
+from coldfront.plugins.qumulo.tasks import reset_allocation_acls
 from coldfront.plugins.qumulo.views.allocation_view import AllocationView
 from coldfront.plugins.qumulo.utils.acl_allocations import AclAllocations
 from coldfront.plugins.qumulo.utils.active_directory_api import ActiveDirectoryAPI
 
 
+logger = logging.getLogger(__name__)
+
 class UpdateAllocationView(AllocationView):
     form_class = UpdateAllocationForm
-    template_name = "allocation.html"
+    template_name = "update_allocation.html"
```

Comment:
@ris-bmulligan is this related to the reset acls?

Reply:
The reset ACLs UI components go with the update allocation form and view.  They are intended to not display in the create allocation form.